xapian-core/tests/termgentest.cc

```diff
@@ -50,7 +50,7 @@ struct test {
     //    (this persists for subsequent tests until it's turned off).
     //  - some: Set stemming strategy to STEM_SOME.
     //    (this persists for subsequent tests until it's turned off).
-    //	- stop=FOO Change stemming algorithm to foo ("none" to turn off).
+    //	- stop=en: Add the stop words.
```

Comment:
I'd state the stop words explicitly, or perhaps "Add a small set of English stop words".